addons/mail/static/src/components/activity_mark_done_popover/activity_mark_done_popover.scss

```diff
@@ -5,7 +5,7 @@
 .o_ActivityMarkDonePopover {
     align-content: space-between;
     display: flex;
-    flex-flow: row wrap;
+    flex-flow: column;
```

Comment:
Good catch, `row wrap` was definitely wrong, but I don't think we need flex column to put 2 div one below the other, can't we just remove all these rules instead?

Reply:
Indeed, also did the same with buttons div as flex was not needed neither